paho/client.go

```diff
@@ -377,7 +377,7 @@ func (c *Client) writer() {
 		}
 		_, err := w.WriteTo(c.Conn)
 		if err != nil {
```

Comment:
Perhaps do we want to skip `context canceled` errors?
Or at least don't log them with error level.

Reply:
done 🙋